### DISCUSSION TITLE
Help, please! The respond pane doesn't appear

### DISCUSSION THREAD

**Discussion by eshatokhin** [Huachao/vscode-restclient#784] [Q&A]
Help, please

I've installed REST-client, all works fine, but after clicking "Send request" the respond pane doesn't appear.
I tried to reinstall clean vscode with only one extension REST-client, but the respond pane doesn't appear.

**Comment by Huachao** [comment]
@eshatokhin did this still happen?

**Reply by eshatokhin** [reply]
Thanks, I already solved this problem, there was mistake in my code. Extension works properly.

**Reply by jinyk** [reply]
When the "request" code is wrong, where do errors show up? I'm sure there is some error and that's why the response pane doesn't appear, but I don't see any errors in output, debug console, or anywhere else. Any idea?

**Comment by runoncedev** [comment]
> When the "request" code is wrong, where do errors show up? I'm sure there is some error and that's why the response pane doesn't appear, but I don't see any errors in output, debug console, or anywhere else. Any idea?

I usually get a VSCode pop-up notification but it would be awesome if it would print into the console or something like that.
